tests/integration/test_integration_platform_doc.py

```
@@ -1145,73 +1211,263 @@ def test_complete_workflow(self, local_docs_dir, temp_dir, agent_config):
-        logger.info(f"Initialized {init_result.total_chunks} chunks from {init_result.total_docs} docs")
+        # Verify chunk content
+        for chunk in doc_result.chunks[:3]:
+            assert "chunk_text" in chunk
+            assert chunk["chunk_text"]
+            logger.info(f"  Chunk: {chunk.get('title', 'N/A')[:50]}...")
 
-        # Step 2: Create SearchTool
-        class TestConfig:
-            def document_storage_path(self, platform):
-                return db_path
+    def test_complete_workflow_local_multi_dir(self, temp_dir, agent_config):
```

Ruff (0.15.0)

[warning] Unused method argument: `agent_config`

(ARG002)